### PR TITLE
feat(entities-plugins): add confluent plugin

### PR DESCRIPTION
**Pull request by sumimakito**
# Summary

Add the Confluent plugin

Gateway PR: https://github.com/Kong/kong-ee/pull/9947

- [x] Schema & overrides
- [x] Icon (Using logo: better to have a proper icon)
- [x] Description
- [x] Category
- [x] Tier
- [x] Adoption & tests: KM

KM-418

### CODE REVIEW

**Review by nekolab — DISMISSED** [review]
LGTM

File: packages/entities/entities-plugins/src/definitions/metadata.ts [hunk 1]
```diff
@@ -748,6 +748,14 @@ export const PLUGIN_METADATA: Record<string, Omit<PluginMetaData<I18nMessageSour
     scope: [PluginScope.GLOBAL, PluginScope.SERVICE, PluginScope.ROUTE, PluginScope.CONSUMER, PluginScope.CONSUMER_GROUP],
     imageName: 'oauth2',
   },
+  'confluent': {
+    descriptionKey: 'plugins.meta.confluent.description',
+    group: PluginGroup.TRANSFORMATIONS,
+    isEnterprise: true,
+    nameKey: 'plugins.meta.confluent.name',
+    scope: [PluginScope.GLOBAL, PluginScope.SERVICE, PluginScope.ROUTE, PluginScope.CONSUMER],
+    imageName: 'confluent',
```
Comment (Leopoldthecoder) [inline packages/entities/entities-plugins/src/definitions/metadata.ts:757]: nit: if the plugin name and the icon's file name are the same, `imageName` can be removed.

Comment (sumimakito) [reply packages/entities/entities-plugins/src/definitions/metadata.ts]: Nice catch. I've updated the pull request

**Review by Leopoldthecoder — APPROVED** [review]
LGTM

**Review by nekolab — APPROVED** [review]

**Pull request merged**